frontend/package.json

```diff
@@ -61,6 +61,7 @@
     "deck.gl": "^8.2.5",
     "dompurify": "^2.2.8",
     "enzyme": "^3.11.0",
+    "eslint-plugin-no-relative-import-paths": "^1.5.0",
```

Comment:
I think this should be added as a dev dependency rather than a normal dependency

Reply:
It's moved to dev dependency 👍